components/Navigation.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+import Image from "next/image";
+import homeIcon from "../public/images/homeIcon.png";
+import addIcon from "../public/images/addIcon.png";
+import { useRouter } from "next/router";
+
+export default function Navigation() {
+  const router = useRouter();
+  return (
+    <StyledNavigation>
+      <UlStyledList>
+        <StyledList>
+          <SyledHomeButton onClick={() => router.push("/")} type="button">
+            <Image src={homeIcon} alt="Go to Homepage" />
+          </SyledHomeButton>
+        </StyledList>
+        <StyledList>
+          <StyledAddButton onClick={() => router.push("Create")} type="button">
+            <Image src={addIcon} alt="Go to Createpage" />
+          </StyledAddButton>
```

Comment:
Why do you use buttons for this? Tip: `next/link`

Reply:
> Why do you use buttons for this? Tip: `next/link`

I will look into `next/link`  :eyes: